iso_week_test.go

```diff
@@ -4,11 +4,17 @@ import (
 	"testing"
 	"time"
 
-	"bitbucket.org/splice/api/models"
+	"bitbucket.org/splice/api/splice/logger"
```

Comment:
booh me, I removed the models one to add the logger :(  Thanks for the great catch